src/ProjectTemplates/Web.ProjectTemplates/content/EmptyComponentsWebAssembly-CSharp/.template.config/template.json

```diff
@@ -205,17 +205,13 @@
       "displayName": "_Progressive Web Application",
       "description": "If specified, produces a Progressive Web Application (PWA) supporting installation and offline use."
     },
-    "RequiresHttps": {
-      "type": "computed",
-      "value": "(!NoHttps)"
-    },
     "HasHttpProfile": {
       "type": "computed",
-      "value": "(!RequiresHttps)"
+      "value": "(NoHttps)"
```

Comment:
The [current `blazorwasm` template only disables the HTTP profile](https://github.com/dotnet/aspnetcore/blob/1cc09115c8d64a17d9cc71972d065bf0f6fdaaab/src/ProjectTemplates/Web.ProjectTemplates/content/ComponentsWebAssembly-CSharp/.template.config/template.json#L560-L567) in the event that we have `Individual/Org` Auth. In the empty template case, given this isn't a factor, I believe we should always keep the `http` profile.

So instead of a `computed` property here, we'd have a:

```json
      "type": "parameter",
      "datatype": "bool",
      "defaultValue": "true",
      "description": "Always have HTTP profile."
```

Reply:
If the HTTP profile is only disabled with authentication, then it makes sense for the empty template to have this enabled by default. I have changed the property type to parameter with the above changes.